### ISSUE TITLE
empty Response in case of invalid Token

### ISSUE THREAD

**Issue by digitalkaoz** [lexik/LexikJWTAuthenticationBundle#124]
if im sending an invalid token i get an empty response, but i would expect some "Invalid Credentials" Response like i get when there is no token provided.

any idea what might be wrong i dont see an obvious error in my app...authentication works

**Comment by SmurfyFR** [comment]
I'm getting the same problem here too.

**Comment by slashfan** [comment]
Hi, 

It is indeed not the expected behavior. Have you looked at the [sandbox](https://github.com/slashfan/LexikJWTAuthenticationBundleSandbox) ? Specifically the [CLI testing section](https://github.com/slashfan/LexikJWTAuthenticationBundleSandbox#manually-check-token-generation-using-the-cli) ? If i submit a wrong password I receive a 401 with a code and message in the response body.

Could you show us your `security.yml ` ? Along with Symfony version and JWT bundle version you are using ?

**Comment by digitalkaoz** [comment]
Hey using Symfony 3.0.1, 1.4.2 of your bundle, security looks as follows:

```yml
security:
    providers:
        in_memory:
            memory:
              users:
                  %api.user%:
                      password: %api.password%
                      roles: 'ROLE_USER'
    encoders:
        Symfony\Component\Security\Core\User\User: plaintext

    firewalls:
        dev:
            pattern: ^/(_(profiler|wdt)|css|images|js)/
            security: false

        doc:
            pattern: ^/$
            security: false

        token:
            pattern:  ^/token$
            stateless: true
            gfreeau_get_jwt: ~

        api:
            pattern:   ^/
            stateless: true
            lexik_jwt:
                query_parameter:
                    enabled: true
                    name:    token

    access_control:
        - { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY }
        - { path: ^/_profiler, roles: IS_AUTHENTICATED_ANONYMOUSLY }
        - { path: ^/,       roles: IS_AUTHENTICATED_FULLY }

```

**Comment by slashfan** [comment]
Hi @digitalkaoz did you find the cause of the problem ?

**Comment by digitalkaoz** [comment]
Nope

Nicolas Cabot <notifications@github.com> schrieb am Mi., 27. Jan. 2016
18:57:

**Comment by chalasr** [comment]
@slashfan To reproduce the problem, you need to send an invalid token, not submit a bad password. Try to make an authenticated request, and for example remove one or more characters in your token. You'll get an empty Response. Fixed in #142

**Comment by chalasr** [comment]
Fixed by PR #142
